R/scmake.R

```diff
@@ -351,6 +351,33 @@ combine_to_ind <- function(ind_file, ...){
   sc_indicate(ind_file = ind_file, data_file = c(...))
 } 
 
+#' Create an object that contains hashes of other files
+#'
+#' @param ... files to combine into a single indicator file
+#' 
+#' @details light wrapper on `sc_indicate`
+#' @export
+#' @examples 
+#' tfiles <- tempfile(pattern=as.character(1:3))
+#' file.create(tfiles)
+#' combine_to_tibble(tfiles[1])
+#' combine_to_tibble(tfiles[1:2])
+#' combine_to_tibble(tfiles[1:3])
+combine_to_tibble <- function(...){
```

Comment:
Interesting and more comprehensive solution than what I did quickly [here](https://github.com/collnell/gage_age_gif/pull/1/files#diff-a3f350b879f5de0415f9e1c1655b6cabR18).

I see good reason to use `sc_indicate` for all file hashing activities, but the file length differences does require some odd work arounds. Good solution though :+1:

This looks good but does bring up the question on whether we want a few more tests in this package as a future goal? In this case, I would see verifying that `combine_to_tibble(tfiles[1]) == combine_to_tibble(tfiles)[1,]` is TRUE

Reply:
Hah, good point that we could just bypass `sc_indicate`...I think I'll stick with it but agree it's clunkier in some ways.

I'll add a test.